transport-native-epoll/src/main/java/io/netty/channel/epoll/Native.java

```diff
@@ -39,7 +39,7 @@
  *
  * <strong>Internal usage only!</strong>
  */
-final class Native {
+public final class Native {
```

Comment:
Does it make sense to extract the "common" items which require this to be public to somewhere more accessible? I thought we wanted to separate `epoll` and `unix`?